### ISSUE TITLE
Add Server Name to Push Notifications

### ISSUE THREAD

**Issue by graywolf336** [RocketChat/Rocket.Chat.Cordova#47]
Add the name of the server where the push notification is from on the notification, this will be extremely useful when you have several instances you're connected to and people have the same usernames.
##

**Comment by geekgonecrazy** [comment]
:+1:

**Comment by rodrigok** [comment]
I agree, but this will be truth to who has only one instance too, cuz we need to do this at the server side.
